luda-editor/luda-editor-client/src/app/sequence_list/types/sequence_index.rs

```diff
@@ -0,0 +1,122 @@
+use linked_hash_map::LinkedHashMap;
+use luda_editor_rpc::Socket;
+
+const SEQUENCE_INDEX_PATH: &str = "sequence_index.json";
+
+pub struct SequenceIndex {
+    sequences: Vec<String>,
```

Comment:
let's name it `sequence_title` because there are `Sequence` so it would be confusing

Reply:
You mean `SequenceIndex.sequences` to `SequenceIndex.sequence_titles`?
Or `SequenceIndex` to `SeuqenceTitles` and `sequence_titles.json`?

Reply:
SequenceIndex.sequence_titles << this